tendrl/notifier/notification/handlers/mail_handler.py

```diff
@@ -86,29 +107,11 @@ def format_message(self, alert):
         return "Subject: [Alert] %s, %s threshold breached\n\n%s" % (
             alert.resource, alert.severity, alert.tags['message'])
 
-    def get_alert_destinations(self, alert):
```

Comment:
Why this method is removed?

Reply:
renamed this function as populate_alert_destinations() as per @shtripat comment https://github.com/Tendrl/notifier/pull/101#discussion_r134691614

Reply:
@GowthamShanmugam The whole point having the plugin framework is to provide a uniform interface. There is no point in defining custom functions when similar things exists in the framewoek; you can simply override them. This will make more sense when you support more  alert delivery mechanisms like SNMP and that's the reason why Anmol came yup with something like this.

@anmolbabu , comments?

Reply:
@nthomas-redhat Yes that's correct but I guess what @GowthamShanmugam is saying is, he has renamed the function both in this plugin and the plugin base class so its basically just a rename as @GowthamShanmugam probably suggests. But @GowthamShanmugam Why the rename in https://github.com/Tendrl/notifier/pull/101/files#diff-ff8fddac84def8409a8553854e56602aR45 was not done?

Reply:
@anmolbabu i will change in base class also

Reply:
@GowthamShanmugam , chnage wherever applicable. As much as possible align with what is defined in the base

Reply:
@nthomas-redhat ok